README.md

```diff
@@ -182,7 +182,7 @@ vendor_package:
   - name: lowrisc_opentitan
     # target directory
     target_dir: vendor/lowrisc_opentitan
-    # upstream dependency (i.e. git repository similar to dependencies)
+    # upstream dependency (i.e. git repository similar to dependencies, but does not support version)
```

Comment:
It also does not support branch names, so it might make more sense to state
```suggestion
    # upstream dependency (i.e. git repository similar to dependencies, only supports commit hash)
```